ts/components/LandingCard.tsx

```diff
@@ -27,17 +27,20 @@ const style = StyleSheet.create({
     width: screenWidth / 2,
     height: screenWidth / 2,
     resizeMode: "contain"
+  },
+  text: {
+    fontSize: 20
   }
 });
 
 export const LandingCardComponent: React.SFC<Props> = card => (
```

Comment:
the file name is different from the item component. What about make it to be the same?

Reply:
Ok :)